conf/aggregations.go

```diff
@@ -85,6 +91,38 @@ func ReadAggregations(file string) (Aggregations, error) {
 			}
 		}
 
+		writeBufferStr := s.ValueOf("writeBuffer")
```

Comment:
where is this format documented? there's 2 files to update:
```
./scripts/config/storage-aggregation.conf
./docker/docker-dev-custom-cfg-kafka/storage-aggregation.conf
```
also run scripts/config-to-doc.sh to update docs/config.md